components/Slider/examples/SliderCollectionViewController.m

```diff
@@ -100,30 +102,33 @@ - (void)applyModel:(MDCSliderModel *)model {
   _label.text = model.labelString;
   _label.textColor = model.labelColor;
   self.contentView.backgroundColor = model.bgColor;
+  MDCSemanticColorScheme *scheme = [[MDCSemanticColorScheme alloc] init];
+  [MDCSliderColorThemer applySemanticColorScheme:scheme toSlider:_slider];
   _slider.numberOfDiscreteValues = model.numDiscreteValues;
   _slider.value = model.value;
   _slider.filledTrackAnchorValue = model.anchorValue;
   _slider.shouldDisplayDiscreteValueLabel = model.discreteValueLabel;
   _slider.thumbHollowAtStart = model.hollowCircle;
   _slider.enabled = model.enabled;
   _slider.statefulAPIEnabled = YES;
+  
   // Don't apply a `nil` color, use the default
   if (model.sliderColor) {
     [_slider setTrackFillColor:model.sliderColor forState:UIControlStateNormal];
     [_slider setThumbColor:model.sliderColor forState:UIControlStateNormal];
     _slider.inkColor = model.sliderColor;
   }
 
```

Comment:
You will need to explicitly set a default or the reused cells will start "inheriting" the green colors.  This was a bug I found while working on the APIs, but seem to have forgotten to commit the changes.

Reply:
Oh, never mind. the "applyScheme" above should take care of that. Nice!